doc/interfaces/denoise_flow.rst

```diff
@@ -10,43 +10,53 @@ Multiple denoising methods are available in DIPY.
 You can try these methods using your own data; we will be using the data in DIPY.
 You can check how to :ref:`fetch the DIPY data<data_fetch>`.
 
--------------------------
-Denoising using Local PCA
--------------------------
+--------------------------------------
+Denoising using Overcomplete Local PCA
+--------------------------------------
 
-PCA-based denoising algorithms are effective denoising methods because they explore 
-the redundancy of the multi-dimensional information of diffusion-weighted datasets.
+Denoising algorithms based on principal components analysis (PCA) are effective
+denoising methods because they explore the redundancy of the multi-dimensional
+information of diffusion-weighted datasets. The basic idea behind the PCA-based
+denoising algorithms is to perform a low-rank approximation by thresholding the
+eigenspectrum of the noisy signal matrix.
 
-The algorithm to perform the PCA-based denoising involves the following steps:
+The algorithm to perform an Overcomplete Local PCA-based (LPCA) denoising
+involves the following steps:
 
-* First, we estimate the local noise variance at each voxel.
+* Estimating the local noise variance at each voxel.
+* Applying a PCA in local patches around each voxel over the gradient
+directions.
+* Thresholding the eigenvalues based on the local estimate of the noise
+variance, and then doing a PCA reconstruction.
```

Comment:
As you can see below, there is an issue with the rendering. Can you fix it

<img width="821" alt="Capture d’écran 2020-11-10 à 23 24 25" src="https://user-images.githubusercontent.com/23106443/98741441-f19edf00-23ac-11eb-859a-4dbdfd88cd9f.png">

Reply:
Good catch. Sorry. Fixed in ca07e4a.